lib/parameters.py

```diff
@@ -338,6 +338,51 @@ def gen_vp9_vbr_parameters(spec):
   params = gen_vp9_vbr_variants(spec)
   return keys, params
 
+def gen_vp9_cqp_lp_variants(spec):
+  for case, params in spec.iteritems():
```

Comment:
We are working on port python2 to python3. 
For python3 style, Can you try to replace spec.iteritems() with iter(spec.items()) ?